pylops/avo/avo.py

```diff
@@ -299,39 +299,44 @@ def akirichards(theta, vsvp, n=1):
     theta : :obj:`np.ndarray`
         Incident angles in degrees
     vsvp : :obj:`np.ndarray` or :obj:`float`
-        VS/VP ratio
+        :math:`V_S/V_P` ratio
     n : :obj:`int`, optional
-        number of samples (if ``vsvp`` is a scalare)
+        number of samples (if ``vsvp`` is a scalar)
 
     Returns
     -------
     G1 : :obj:`np.ndarray`
         first coefficient of three terms Aki-Richards approximation
-        :math:`[n_{theta}  \times  n_{vsvp}]`
+        :math:`[n_\theta  \times  n_\text{vsvp}]`
     G2 : :obj:`np.ndarray`
         second coefficient of three terms Aki-Richards approximation
-        :math:`[n_{theta}  \times  n_{vsvp}]`
+        :math:`[n_\theta  \times  n_\text{vsvp}]`
     G3 : :obj:`np.ndarray`
         third coefficient of three terms Aki-Richards approximation
-        :math:`[n_{theta}  \times  n_{vsvp}]`
+        :math:`[n_\theta  \times  n_\text{vsvp}]`
 
     Notes
     -----
     The three terms Aki-Richards approximation [1]_ is used to compute the
     reflection coefficient as linear combination of contrasts in
-    :math:`V_P`, :math:`V_S`, and :math:`\rho`. More specifically:
+    :math:`V_P`, :math:`V_S`, and :math:`\rho.` More specifically:
```

Comment:
When the period is outside, the line can break on the period like this:

... and ρ
. More specifically.

Another option is to use the [U+2060 WORD JOINER](https://en.wikipedia.org/wiki/Non-breaking_space#Width_variation) but I thought this might be overkill hahahaha

Reply:
Ah right :)